projects/robots/micromagic/scout/worlds/scout.wbt

```diff
@@ -0,0 +1,840 @@
+#VRML_SIM R2020a utf8
+WorldInfo {
+  info [
+    "A Starwars inspired scout RX28 walking robot"
+    "Author: Matt Denton"
+    "Date: 08 JUN 2020"
+  ]
+  title "scout"
+  basicTimeStep 10
+  FPS 50
+  defaultDamping Damping {
+  }
+}
+Viewpoint {
+  orientation -0.2507087455358656 0.9348870996710782 0.25125929988844664 0.6807068797882645
+  position 0.6306967380708725 0.36415500942140094 0.8138775144579378
+  follow "robot"
+}
+Fog {
+  color 0.8 0.7 0.45
+  fogType "EXPONENTIAL2"
+  visibilityRange 4
+}
+TexturedBackground {
+  texture "mars"
+}
+TexturedBackgroundLight {
+  texture "mars"
+}
+DirectionalLight {
+  ambientIntensity 1
+  direction -0.5 -2 -0.5
+  intensity 0.7
+  castShadows TRUE
+}
+DirectionalLight {
+  ambientIntensity 1
+  direction 0.5 -0.5 0.5
+  intensity 0.7
+  castShadows TRUE
+}
```

Comment:
Do we really need 3 DirectionalLight nodes in the scene?

Reply:
I changed a little bit the lightning so that we now need only 2 directional lights.

Reply:
Perfect!